### PR TITLE
Data : Standardize the error message in TableMigrationUtil

### PR DESCRIPTION
**Pull request by coded9**
- Standardize the error message

### CODE REVIEW

File: data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java [hunk 1]
```diff
@@ -177,7 +177,7 @@ private static Metrics getAvroMetrics(Path path, Configuration conf) {
       long rowCount = Avro.rowCount(file);
       return new Metrics(rowCount, null, null, null, null);
     } catch (UncheckedIOException e) {
-      throw new RuntimeException("Unable to read Avro file: " + path, e);
+      throw new RuntimeException("Unable to read the metrics of the Avro file: " + path, e);
```
Comment (szehon-ho) [inline data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: Hi, thanks for the contribution, I feel the existing message is correct though, as I think we are reading the Avro file, to return metrics from it?

Comment (fivetran-ashokborra) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: Hi @szehon-ho , yes the existing message is correct, but to keep the message consistent similar to Parquet and ORC error messages

Comment (fivetran-ashokborra) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: I can write a common method to keep them consistent which wouldn't require changes at multiple places

Comment (coded9) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: Hi @szehon-ho , the message is correct, but to make it more meaningful and consistent with Parquet and ORC error messages. I can write a common method to accommodate more format, but I feel that is not required at this time.

Comment (szehon-ho) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: Hi, I believe the problem is, though, Avro file does not have metrics per-se.  (Hence we need to read the file to get rowCount).  Versus ORC, Parquet actually have a metrics section of the file.  So I believe the discrepancy in message is originally intentional.  We can check with original author, of course.

Comment (coded9) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: @RussellSpitzer @Fokko

Comment (Fokko) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: You could argue that the row count is also a metric, but I see @szehon-ho's point. I don't have a strong opinion on this one.

Comment (coded9) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: As the method names are in the format - ` get[FileFormat]Metrics` and we are also returning the Metrics object, I don't see like we are deviating from the context.

Comment (szehon-ho) [reply data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java:180]: Sure, was just pointing out the possible intent, Fokko's argument also makes sense, and I dont have a strong opinion either.

**Pull request closed without merging**